### PR TITLE
change directory layout to suit PHPCS installed_paths feature, fixes #46

### PR DESCRIPTION
**Pull request by xalopp**

### CODE REVIEW

**Comment by xalopp** [comment]
@mmoll could you have a look at this PR. I guess there are still some typos left :-)
Would be great if you can check if the Instructions in README and CONTRIBUTING are correct.

Shall we do a v1.0.0  or v0.1.0 release with this PR?
And should we make a legacy release? (v0.0.1) In this case, a I would add a CHANGES.md / CHANGELOG.md /... file to help migration 😄

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,29 +5,24 @@
...
 If you contribute code, please make sure it conforms to the PHPCS coding standard and that the unit tests still pass.
 
-1. To check the coding standard, run in the repository root of `PHP_CodeSniffer`:
+1. To check the coding standard, run in the repository root:
 
-        ./bin/phpcs --ignore='*/vendor/*' src/Standards/MO4
+        ./vendor/bin/phpcs MO4
 
-2. The unit-tests are run from within the `PHP_CodeSniffer` directory
+2. To check the unit tests, run in the repository root:
```
Comment (mmoll) [inline CONTRIBUTING.md]: "To run the unit tests, execute:"

File: README.md [hunk 2]
```diff
@@ -26,27 +26,78 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
...
+### Composer
 
-        pear install PHP_CodeSniffer
+Using [Composer](https://getcomposer.org/) is the preferred way.
+
+1. Add the MO4 Repository
+
+        composer config repositories.mo4 vcs https://github.com/mayflower/mo4-coding-standard.git
+
+If composer complains about a missing composer.json, just create a empty one:
```
Comment (mmoll) [inline README.md]: "..., create an empty one"

File: README.md [hunk 3]
```diff
@@ -26,27 +26,78 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
 
 ## Installation
 
-1. Install phpcs:
+### Composer
 
-        pear install PHP_CodeSniffer
+Using [Composer](https://getcomposer.org/) is the preferred way.
+
+1. Add the MO4 Repository
```
Comment (mmoll) [inline README.md]: repository all lower case 🇬🇧

File: README.md [hunk 4]
```diff
@@ -26,27 +26,78 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
...
+
+        composer config repositories.mo4 vcs https://github.com/mayflower/mo4-coding-standard.git
+
+If composer complains about a missing composer.json, just create a empty one:
+
+        echo '{}' > composer.json
+
+2. Add the MO4 Coding Standard to `composer.json`
```
Comment (mmoll) [inline README.md]: "coding standard" lower case 🇬🇧

File: README.md [hunk 5]
```diff
@@ -26,27 +26,78 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
...
+
+        ./vendor/bin/phpcs path/to/my/file.php
+
+or
+
+        ./vendor/bin/phpcs --standard=MO4 path/to/my/file.php
+
+### Source
+
+1. Checkout the this repository
```
Comment (mmoll) [inline README.md]: -the

File: README.md [hunk 6]
```diff
@@ -26,27 +26,78 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
...
+
+If composer complains about a missing composer.json, just create a empty one:
+
+        echo '{}' > composer.json
+
+2. Add the MO4 Coding Standard to `composer.json`
+
+        composer require --dev mayflower/mo4-coding-standard:dev-master
+
+3. Optionally, you might set MO4 as default coding standard:
```
Comment (mmoll) [inline README.md]: either finish all these lines with a colon or none

File: README.md [hunk 7]
```diff
@@ -26,27 +26,78 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
...
 
-3. Copy, symlink or check out the Symfony coding standard and this repository to their respecting folders inside the
-   phpcs `Standards` directory:
+        composer install
 
-        cd /path/to/pear/PHP/CodeSniffer/src/Standards
-        git clone https://github.com/djoos/Symfony-coding-standard.git Symfony
-        mv Symfony/Symfony/* Symfony/
-        git clone https://github.com/mayflower/mo4-coding-standard.git MO4
+3. Check that Symfony and MO4 are listed as coding standards:
```
Comment (mmoll) [inline README.md]: see above regarding colons

Comment (mmoll) [reply README.md]: "Check, that..."

File: README.md [hunk 8]
```diff
@@ -26,27 +26,78 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
...
+        git clone https://github.com/djoos/symfony-coding-standard.git
+        git clone https://github.com/mayflower/mo4-coding-standard.git
+
+3. Select the MO4 ruleset as your default coding standard:
+
+        phpcs --config-set installed_paths PATH/TO/symfony2-coding-standard,PATH/TO/mo4-coding-standard
         phpcs --config-set default_standard MO4
 
-5. Profit
+4. Profit
```
Comment (mmoll) [inline README.md]: colon?

File: composer.json [hunk 9]
```diff
@@ -35,12 +21,11 @@
         "issues": "https://github.com/mayflower/mo4-coding-standard/issues",
         "source": "https://github.com/mayflower/mo4-coding-standard"
     },
-    "require-dev": {
```
Comment (mmoll) [inline composer.json]: wait, why? 😳

**Comment by mmoll** [comment]
Overall this is a terrific improvement, which eases installation and usage of ths CS vastly. I expect this to help boosting the adoption rate tremendously!

File: README.md [hunk 10]
```diff
@@ -26,28 +26,79 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
...
+
+3. Optionally, you might set MO4 as default coding standard
+
+        ./vendor/bin/phpcs --config-set default_standard MO4
+
+4. Profit
+
+        ./vendor/bin/phpcs path/to/my/file.php
+
+or
```
Comment (mmoll) [inline README.md]: the alignment in the rendered document (at least when viewing on GH) is off here and in two similar occurrences

File: README.md [hunk 11]
```diff
@@ -26,37 +26,83 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
...
+### Composer
 
-        pear install PHP_CodeSniffer
+Using [Composer](https://getcomposer.org/) is the preferred way.
+
+1. Add the MO4 repository
+
+        composer config repositories.mo4 vcs https://github.com/mayflower/mo4-coding-standard.git
+
+If composer complains about a missing composer.json, just create an empty one
```
Comment (mmoll) [inline README.md]: that block is still off

**Comment by mmoll** [comment]
merci @xalopp!

**Pull request merged**